mtbf_driver/mtbf.py

```diff
@@ -205,16 +206,29 @@ def deinit(self):
         if os.path.exists(info):
             shutil.copy2(info, self.workspace)
 
+        archive_file = os.path.basename(self.archive_folder) + ".zip"
+        try:
+            os.remove(archive_file)
```

Comment:
Should we try to use different archive file name instead of killing the old one? Let's say... If a partner start mtbf_operation for multiple times, it might have some issues?